.tt_blacklist

```diff
@@ -1,6 +1,4 @@
 chemicaltoolbox/osra
-chemicaltoolbox/silicos-it/shape-it
```

Comment:
At least the packages you remove should be triggered to test.
In a separate PR it is easier for you to track errors.